INSTALL_ISSUES.md

```diff
@@ -22,3 +22,25 @@ ___
 `ImportError: liblapack.so.3: Cannot open shared object file: No such file or directory`
 
 You need to install Blas and Lapack. See [LINUX\_INSTALL.md](LINUX_INSTALL.md) for details.
+
+___
+
+`ModuleNotFoundError: No module named 'turicreate'` with Jupyter Notebook
```

Comment:
When users are having this issue, I recommend that they just install turicreate from inside of Jupyter Notebooks. I think this is approach is simpler. My instructions fo that are [here](https://github.com/apple/turicreate/issues/3196#issuecomment-629832556). I think it might make more sense to just document this approach. @ornob011 - what do you think?

Reply:
I think you're right. Document my approach, and say that my approach should only be tried if no other approach is working.